Updating details for support of DLP policy tips @chrfox (#4529)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/use-notifications-and-policy-tips.md

You can customize the text for policy tips separately from the email notification. Unlike custom text for email notifications (see above section), custom text for policy tips does not accept HTML or tokens. Instead, custom text for policy tips is plain text only with a 256-character limit.
  

> ## Support Matrix for DLP policy tips across Microsoft apps

Reviewer: ```suggestion
## Support matrix for DLP policy tips across Microsoft apps
```
```suggestion
## Support Matrix for DLP policy tips across Microsoft apps
```

> ## Support Matrix for DLP policy tips across Microsoft apps

Reviewer: ```suggestion
## Support matrix for DLP policy tips across Microsoft apps
```
```suggestion
## Support Matrix for DLP policy tips across Microsoft apps
```

|**App and platform**|**DLP policy tip support**|**Sensitive information types supported**|**Conditions and exceptions supported**|**Comments**|
|--------------------|--------------------------|-----------------------------------------|------------------------------------|------------|

> |Outlook Web Access <br/> |Yes <br/> |All <br/> |Subset  <br/> |See  [DLP predicates supported for DLP policy tips in Outlook Web Access](#dlp-predicates-supported-for-dlp-policy-tips-in-outlook-web-access)  <br/> |

Reviewer: ```suggestion
|Outlook on the web <br/> |Yes <br/> |All <br/> |Subset  <br/> |See  [DLP predicates supported for DLP policy tips in Outlook on the web](#dlp-predicates-supported-for-dlp-policy-tips-in-outlook-web-access)  <br/> |
```
```suggestion
|Outlook Web Access <br/> |Yes <br/> |All <br/> |Subset  <br/> |See  [DLP predicates supported for DLP policy tips in Outlook Web Access](#dlp-predicates-supported-for-dlp-policy-tips-in-outlook-web-access)  <br/> |
```

|Outlook Desktop (Outlook 2013 and beyond) <br/> |Yes <br/> |Subset <br/> |Subset  <br/> |See [Outlook 2013 and later supports showing policy tips for only some sensitive information types](#outlook-2013-and-later-supports-showing-policy-tips-for-only-some-sensitive-information-types) and [Outlook 2013 and later supports showing policy tips for only some conditions and exceptions](#outlook-2013-and-later-supports-showing-policy-tips-for-only-some-conditions-and-exceptions) for details on support for sensitive information types and DLP conditions and actions supported for showing DLP policy tips on Outlook Desktop. <br/> |

> |Outlook Mobile (iOS, Android)/Outlook Mac <br/> |No <br/> |None <br/> |None <br/> |DLP policy tips are not supported on Outlook mobile <br/> |

Reviewer: ```suggestion
|Outlook for mobile <br/> |No <br/> |None <br/> |None <br/> |DLP policy tips are not supported on Outlook for mobile <br/> |
```
